app/furniture/marketplace/marketplaces_controller.rb

```diff
@@ -9,5 +9,22 @@ def show
     def edit
       authorize(marketplace)
     end
+
+    def update
+      if authorize(marketplace).update(marketplace_params)
+        redirect_to marketplace.location(child: :notification_methods), notice: t(".success", name: marketplace.room.name)
+
+        if marketplace_params[:square_access_token] || marketplace_params[:square_location_id]
+          flash.notice = "Square notification settings updated succesfully!"
+          flash.alert = "Square notification settings were not upated. Please try again or contact your site administrator."
+        end
```

Comment:
Will add template tag to render flash messages within the turbo'd page scope later.

Reply:
Why are the notice and alert in the same `if`?

Reply:
My goal here was to set custom message text for _any_ interaction with the Square settings.

Reply:
Oh oops, I actually got confused. This was a mistake 😅

Reply:
Well, it will definitely set a custom message for any interaction! They'll just be... a bit contradictory. That's all!